package.json

```diff
@@ -56,8 +57,5 @@
     "webpack-stream": "^3.1.0",
     "yargs": "^4.4.0"
   },
-  "dependencies": {},
-  "publishConfig": {
```

Comment:
It looks like the publishConfig section was removed during some merge.  I think this section should stay.